### PR TITLE
Move unsorted channels to their respective countries

### PR DESCRIPTION
**Pull request by BellezaEmporium**
This PR is a placeholder for unsorted channels.

In fact, among all channels we got, there's lots of duplicates that are in unsorted. Mainly a source of backup channels we could use.

Discussions on some of my choices are appreciated.

Duplicate streams if kept will follow the new and discussed naming convention (see #1916)

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
First part of unsorted ready for review.

File: channels/ve.m3u
Comment (themak1985) [inline channels/ve.m3u:6]: remove bitrate/backup text

File: channels/ve.m3u
Comment (themak1985) [inline channels/ve.m3u:8]: remove bitrate/backup text

File: channels/ve.m3u
Comment (themak1985) [inline channels/ve.m3u:11]: Add a blank line at end of file

File: package.json
Comment (themak1985) [inline package.json:14]: ?

File: package-lock.json
Comment (themak1985) [inline package-lock.json:7]: ?

**Review by themak1985 — COMMENTED** [review]
Hey @LaneSh4d0w great work, but let's hold off on adding bitrate information and let's avoid adding (backup) as it can become a maintenance issue. 

Also, not sure why there are changes to languages.md, readme.md, package.json, and package-lock.json but I will defer to @freearhey on those changes.

**Comment by freearhey** [comment]
@themak1985 As for `_categories.md`, `_countries.md`, `_languages.md`, `README.md`  they are all generated automatically every day and there is no need to update them manually. 

As for updating `axios` version, there's nothing wrong with that, I just don't understand what it has to do with the subject of this PR.

**Comment by LaneSh4d0w** [comment]
Ah yes, I think i had updated the readme mistakenly, i'll copy and paste the data from master.

File: .readme/_categories.md
Comment (LaneSh4d0w) [reply .readme/_categories.md:7]: It was the work of the script to be honest.

File: channels/bo.m3u
Comment (LaneSh4d0w) [reply channels/bo.m3u:4]: Well, we do not know the location of this ATB. (At the moment I was watching, they were all 3 (La Paz, Cochabamba and Santa Cruz) streaming a Disney program for kids. Since I do not know the exact location of this, I preferred adding a backup.

File: package.json
Comment (LaneSh4d0w) [reply package.json:14]: If I remember well, @freearhey got the same changes.

File: package-lock.json
Comment (LaneSh4d0w) [reply package-lock.json:7]: Package changes.

File: channels/ht.m3u
Comment (freearhey) [inline channels/ht.m3u:7]: Why did you replace the correct logo and name with the wrong one?

File: channels/int.m3u
Comment (freearhey) [inline channels/int.m3u:85]: Hustler is US channel: https://www.lyngsat.com/tvchannels/us/Hustler-HD-Europe.html

File: channels/us.m3u
Comment (freearhey) [inline channels/us.m3u:934]: this line must be deleted

File: channels/us.m3u
Comment (freearhey) [inline channels/us.m3u:936]: this line must be deleted

File: channels/us.m3u
Comment (freearhey) [inline channels/us.m3u:940]: this line must be deleted

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/ht.m3u
Comment (LaneSh4d0w) [reply channels/ht.m3u:7]: Ah ! Well... x) i'm guilty for this one !

File: channels/int.m3u
Comment (LaneSh4d0w) [reply channels/int.m3u:85]: Noted

File: channels/us.m3u
Comment (LaneSh4d0w) [reply channels/us.m3u:934]: Git doing things... Noted

File: channels/us.m3u
Comment (LaneSh4d0w) [reply channels/us.m3u:940]: That's because I deleted the City name before the actual Channel name I suppose.

**Review by BellezaEmporium — COMMENTED** [review]
Ready for second review.

**Review by freearhey — APPROVED** [review]

**Pull request merged**
